projects/isy-angular-widgets/src/lib/form-wrapper/form-wrapper.component.spec.ts

```diff
@@ -64,20 +64,16 @@ describe('FormWrapperComponent', () => {
     ).toThrowError('control Input is required and must be an instance of FormControl');
   });
 
-  it('label should not include a "*" by default if field is optional (non required)', () => {
-    spectator.component.validationMessages = {key: 'non_required'};
-    spectator.detectChanges();
```

Comment:
A new test has to be added:
label should not include an asterisk "*" by default